compiler/msgs.nim

```diff
@@ -64,6 +64,8 @@ type
     errVarForOutParamNeeded,
     errPureTypeMismatch, errTypeMismatch, errButExpected, errButExpectedX,
     errAmbiguousCallXYZ, errWrongNumberOfArguments,
+    errWrongNumberOfArgumentsInCall,
```

Comment:
> Under the hood, this may even revive your initial vision of a localized compiler 

I don't want a localized compiler anymore. Harder to google for error messages, errors coming from macros would also need to be translated or else you get a mix of different languages...